client/blocks/design-sidebar/index.jsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import RootChild from 'components/root-child';
+
+class DesignSidebar extends Component {
+	constructor( props ) {
+		super( props );
+	}
+
+	render() {
+		const className = 'design-sidebar' + ( this.props.isVisible ? ' is-visible' : '' );
```

Comment:
might as well use `classNames` here since we use it all over the place

```js
const className = classNames( 'design-sidebar', { 'is-visible': this.props.isVisible } );
```